lib/galaxy/datatypes/last.py

```diff
@@ -0,0 +1,18 @@
+from galaxy.datatypes.data import Data
+
```

Comment:
@RomainDallet should be move this into the blast.py file? The tools are somehow connected ...